whois-api/src/main/java/net/ripe/db/whois/api/nrtm4/NrtmClientService.java

```diff
@@ -103,6 +106,12 @@ private void validateSource(final String source, final String fileName) {
         }
     }
 
+    private void validateHttps(final HttpServletRequest httpServletRequest) {
+        boolean isHttps = Objects.equals(httpServletRequest.getHeader("X-Forwarded-Proto"), "HTTPS");
```

Comment:
But does the application handles this? I thought we do not allow http requests anyways. F5 does the redirect. And even if applocation needs to handle it should be a requestFilter?

Reply:
In jira ticket:

Implementation Details
All requests come through the F5 LB which adds a X-Forwarded-Proto header. Check the value of this header, if it's set to "http" then return a 426 response.

Reply:
This should be job of  request filter. Can you create a filter instead and add it to WhoisNrtmServletDeployer ?

Reply:
ok, let me check this